core/server/data/migrations/versions/3.25/02-add-amp-gtag-id-setting.js

```diff
@@ -0,0 +1,20 @@
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up({transacting: knex}) {
```

Comment:
I'm not completely sure why we need this migration? Is it to maintain the state migrations produce decoupled form the state of `default-settings.json`?